### DISCUSSION TITLE
Improve linkage to contact tracing people via a chat/messaging interface (still anonymous)

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#51] [Ideas]
# Summary | Résumé
As a user of the app further stressed by the emergence of more viral/lethal strains, I expect diagnosed users reporting back as well as QR code scanning sites reporting back (#1328) to be better linked with people based contact tracing (from a local health authority) so that important outbreak info can be turned around much faster. So far, the app and contact tracing have operated entirely independently and this may no longer be appropriate for new strains and virus requiring a stronger response. Such a messaging facility may also become vital in the event that traditional contact tracing collapses under too many cases. 

I expect to be able to maintain anonymousness with the contact tracer at times or sometimes be willing to further engage elsewhere disclosing more personal info (e.g. phone number, email, text messaging, Google map trace, app log including my proximity contacts cds-snc/covid-alert-app#1304 and QR code site contacts cds-snc/covid-alert-documentation#44, etc...).  A minimum chat/messaging interface could make a huge difference for making the app more useful, boost adoption/retention rates and facilitate the work of contact tracing people. 

# Design detail | Précisions au sujet de la conception
The contact tracer could initiate the chat/messaging exchange (following a positive diagnostic) by sending a new type of alert notification (-vs- diagnosed user exposure and diagnosed QR code site exposure) upon which the app user would preferably respond and engage in chat/messaging. Alternatively, the app user may wish to initiate contact with contact tracing people. Personal privacy should remain protected should the app user want. All the messaging data should similarly be removed after 14 days. 

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet
A similar enhancement request should be entered on the portal side component. New types of notifications have not been recognized yet, nor the need to repurpose the app as part of a stronger app response. Further discussions are clearly once again required with the various provincial/local health authorities to better establish related requirements. An advisory council working/steering group should similarly participate.

**Comment by LeCanardQuoi** [comment]
App logged info about  QR scan (s) events cds-snc/covid-alert-documentation#44 or BLE proximity events cds-snc/covid-alert-app#1304 for period of interest could easily be cut&paste in the chat messaging interface used with contact tracing people.

**Comment by LeCanardQuoi** [comment]
The GAEN compliant Kiwi app provides a "Share your digital diary" with a contact tracer (editing of the diary is also allowed). This is the KEY to enabling contact tracing. The #CovidAlert log made of significant app events (e.g. app commands, anonymized proximity contacts and QR scan code contacts) could be similarly shared.
https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-resources-and-tools/nz-covid-tracer-app/getting-started-nz-covid-tracer

**Reply by obrien-j** [reply]
Thanks for the suggestions @LeCanardQuoi ,

I've flagged this idea to the teams working on qr codes. I'm sure this will be top of mind as they continue to discuss with the various P/T's and their contact tracing efforts, to find strategies that work for each individual health authority.

**Reply by LeCanardQuoi** [reply]
Each individual health authority should really be referred to the working model of NZ Tracer app. Provincial  IT/health bureaucracies will otherwise drag this forever, particularly BC. No time left for this. Critical leadership absolutely required, from lessons learnt thus far. Will otherwise repeat earlier mistakes.

**Comment by timarney** [comment]
re: > https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-resources-and-tools/nz-covid-tracer-app/getting-started-nz-covid-tracer

Thanks for the link that's helpful

**Comment by obrien-j** [comment]
In keeping with our newly really release cds-snc/covid-alert-documentation#31  practices, I've moved this up into the covid-alert-documentation repo as our central point for taking input from folks outside the team, and am converting the issue into a Discussion.
